Common/DataModel/TrackSelectionTables.h

```diff
@@ -50,8 +50,9 @@ struct TrackSelectionFlags {
   static constexpr flagtype kDCAxy = 1 << 13;
   static constexpr flagtype kDCAz = 1 << 14;
   // Combo masks
-  static constexpr flagtype kQualityTracks = kTrackType | kTPCNCls | kTPCCrossedRows | kTPCCrossedRowsOverNCls | kTPCChi2NDF | kTPCRefit | kITSNCls | kITSChi2NDF | kITSRefit | kITSHits;
   static constexpr flagtype kQualityTracksITS = kTrackType | kITSNCls | kITSChi2NDF | kITSRefit | kITSHits;
+  static constexpr flagtype kQualityTracksTPC = kTrackType | kTPCNCls | kTPCCrossedRows | kTPCCrossedRowsOverNCls | kTPCChi2NDF | kTPCRefit;
```

Comment:
@mfaggin I defined the QualityTracks from the ITS and TPC ones

Reply:
@mfaggin what do you think?

Reply:
Hi @njacazio sorry I was convinced that my previous reaction was enough. Nothing against it! From the cuts point of view everything seems fine.